mainscript.py

```diff
@@ -93,9 +93,7 @@ def download(link, newdevice, video, delete):
 
     click.secho("File successfully downloaded.", fg="green", bold=True)
     types = ('*.mp3', '*.mp4', '*.mkv')
-    list_of_files = []
-    for files in types:
-        list_of_files.extend(glob.glob(files))
+    list_of_files = list(itertools.chain((glob.glob(ftype) for ftype in types)))
```

Comment:
This gives error when finding the `recent_download`, but my previous declaration of `list_of_files` works perfectly. Could you change it to the previous declaration?